bucket/markdown-monster.json

```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "http://markdownmonster.west-wind.com",
+    "description": "Markdown editing and weblog publishing tool.",
+    "license": "Shareware",
```

Comment:
Missing URL, also I would say this is not shareware

http://markdownmonster.west-wind.com/download.aspx#Licensing
https://markdownmonster.west-wind.com/purchase.aspx#License
https://github.com/RickStrahl/MarkdownMonster/blob/master/LICENSE.md